packages/zoe/src/contracts/otcDesk.js

```diff
@@ -0,0 +1,129 @@
+ * The creator of this contract instance can take three actions: add
+ * inventory, remove inventory, and make quotes for potential trading
+ * partners.
+ *
+ * To add inventory, the creator of the contract instance can call
+ * `E(creatorFacet).makeAddInventoryInvitation(issuerKeywordRecord)`
+ * and receive an invitation to add inventory. In this call, they must
+ * pass in an issuerKeywordRecord of any issuers they wish to add
+ * inventory for, if these issuers have not yet been saved to ZCF.
+ * When actually escrowing the inventory as an offer, the proposal
+ * must not `want` anything. All of the newly escrowed inventory is
+ * taken and reallocated to the marketMakerSeat in the contract.
+ *
+ * To remove inventory, the creator of the contract instance can call
+ * `E(creatorFacet).makeRemoveInventoryInvitation()` and receive an
+ * invitation to remove inventory. When making an offer to remove
+ * inventory, the proposal should specify the `want`, which will be
+ * removed, but should not give anything.
+ *
+ * To make a quote, the creator can call
+ * `E(creatorFacet).makeQuote(price, assets, timeAuthority,
+ * deadline)`. `price` and `assets` are amountKeywordRecords that will
+ * be used in the coveredCall. The assets are the underlyingAssets in
+ * the call option, and the price is used as the strikePrice. The
+ * timeAuthority should be a timer, and the deadline can be any time
+ * understood by the timer. The quote will be cancelled after the
+ * deadline. `makeQuote` returns a covered call option that can be
+ * given away for free or sold. Importantly, if the recipient chooses
+ * to exercise the option, the trade is guaranteed to succeed if their
+ * proposal matches the quote.
+ *
+ * @type {ContractStartFn}
+ */
+const start = zcf => {
+  const { zcfSeat: marketMakerSeat } = zcf.makeEmptySeatKit();
+  const { coveredCallInstallation } = zcf.getTerms();
+  const zoe = zcf.getZoeService();
+
+  const makeQuote = async (price, assets, timeAuthority, deadline) => {
+    const { creatorInvitation } = await E(zoe).startInstance(
+      coveredCallInstallation,
+      zcf.getTerms().issuers,
+    );
+    const shouldBeInvitationMsg = `The covered call instance should return a creatorInvitation`;
+    assert(creatorInvitation, shouldBeInvitationMsg);
+    const proposal = harden({
+      give: assets,
+      want: price,
+      exit: {
+        afterDeadline: {
+          deadline,
+          timer: timeAuthority,
+        },
+      },
+    });
+
+    const payments = await withdrawFromSeat(zcf, marketMakerSeat, assets);
+    const sellerUserSeat = await E(zoe).offer(
+      creatorInvitation,
+      proposal,
+      payments,
+    );
+
+    E(sellerUserSeat)
+      .getPayouts()
+      .then(async payoutPayments => {
+        const amounts = await E(sellerUserSeat).getCurrentAllocation();
+        await depositToSeat(zcf, marketMakerSeat, amounts, payoutPayments);
+      });
+    const option = E(sellerUserSeat).getOfferResult();
```

Comment:
Just return this directly?

Reply:
Hmm I guess there's some value in naming it, but at least delete the intervening white space. But otherwise:
```
// return the option
return E(sellerUserSeat).getOfferResult();
```